### PR TITLE
dns/bind: cleanup master/slave zone UI

### PR DESCRIPTION
**Pull request by sestary**
This cleans up the `master`/`slave` dialog boxes to only contain the items that are actually processed for each zone type. It also remove the Type column from the zone list since this is redundant information.

### CODE REVIEW

**Comment by fichtner** [comment]
sorry about #3182 being merged interfering here... was overlooked previously but seemed better to get in before this one

**Comment by sestary** [comment]
No problem makes sense to get it in there. I've corrected the conflicts that merging the other PR caused. Let me know if anything else needs to be changed.

File: dns/bind/pkg-descr [hunk 1]
```diff
@@ -11,6 +11,7 @@ Plugin Changelog
 
 1.25
 
+* Cleanup/Fix the Master/Slave domain dialogs (contributed by Robbert Rijkse and Michael Newton)
```
Comment (fichtner) [inline dns/bind/pkg-descr:14]:
```suggestion
* Cleanup/Fix the Master/Slave domain dialogs (contributed by Robbert Rijkse)
```

;)

**Comment by fichtner** [comment]
Not sure if I can review this in the next week or so, please ping me in January if I forget. Thanks and happy holidays!

File: dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindSlaveDomain.xml [hunk 2]
```diff
@@ -47,14 +47,14 @@
         <id>domain.transferkey</id>
         <label>Transfer Key</label>
         <type>text</type>
-        <help>The base64-encoded TSIG key.</help>
+        <help>The base64 encoded TSIG key used to transfer domain data from the master server.</help>
```
Comment (fichtner) [inline dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindSlaveDomain.xml:50]:
```suggestion
        <help>The TSIG key used to transfer domain data from the master server in Base64 encoding .</help>
```

File: dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindSlaveDomain.xml [hunk 3]
```diff
@@ -47,14 +47,14 @@
         <id>domain.transferkey</id>
         <label>Transfer Key</label>
         <type>text</type>
-        <help>The base64-encoded TSIG key.</help>
+        <help>The TSIG key used to transfer domain data from the master server in Base64 encoding .</help>
```
Comment (fichtner) [inline dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/dialogEditBindSlaveDomain.xml:50]:
```suggestion
        <help>The TSIG key used to transfer domain data from the master server in Base64 encoding.</help>
```

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**